lib/ex_unit/test/ex_unit/case_test.exs

```diff
@@ -142,6 +142,23 @@ defmodule ExUnit.CaseTest do
                    end
                  end
   end
+
+  test "warns when using @tag outside of describe" do
+    stderr =
+      ExUnit.CaptureIO.capture_io(:stderr, fn ->
+        defmodule TagOutsideOfDescribe do
+          use ExUnit.Case, register: false
```

Comment:
I found this way of fixing test isolation, otherwise would get flaky counts in `ExUnit.RegisterTest`.

Reply:
You can also raise inside the test, so the module is not defined after all.